1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,20 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let num= 0 ;
+  let arr =[]
+  while(n > 0){
+   arr.push(num)
+   num += 2
+   n--
+  }
```

Comment:
This function works. Another way to write it which for me it's a bit more readable would be like this:

`function evenNumbers(n) {
  let num= 0 ;
  let arr =[]
  while(num < n){
   arr.push(2*num)
   num ++
  }`

I find it more natural that the comparison is between n and num but both of them are correct.